### PR TITLE
Hyperledger Indy 2024 Q4 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**

### CODE REVIEW

**Review by WadeBarnes — APPROVED** [review]

File: tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+features in the existing Indy Node/Plenum code base.
+
+The Hyperledger Mentorship project has produced an [Indy Read Replica]
+implementation. We expect to have a
+[repository](https://github.com/rxbryan/indyread) containing a first cut of the
+functionality and documentation completed in November.
+
+[Indy Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=indy&routedFrom=Github&bestPractice=false&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true
+
+## Questions/Issues for the TOC
```
Comment (tkuhrt) [inline tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md:71]:
```suggestion
## Questions/Issues for the TAC
```

File: tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+- indy-plenum - v1.14.0rc0 -- included in the last Quarterly report, but occurred in the period covered by this report.
+- indy-vdr - v0.4.3 -- included in the last Quarterly report, but occurred in the period covered by this report.
+
+## Overall Activity in the Past Quarter
+
+Work was done this quarter on the Ubuntu 22.04 version of Indy, with a PR
+submitted for Indy Node in July. However, that PR has not been approved/merged
+into the codebase. That has prevented the completion of a release candidate for
+the Indy on Ubuntu 22.04.
```
Comment (tkuhrt) [inline tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md:93]: Is the PR not being approved or merged due to lack of maintainers? Or is there another reason? Do we need to be concerned?

Comment (petermetz) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md:93]: @tkuhrt I tracked the PR [1] down and it does look a little concerning: Created 7 months ago, reviewers added 4 and 3 months ago, but no response to it so far. :/

[1] https://github.com/hyperledger/indy-node/pull/1870

File: tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md (hunk 1 above)
Comment (swcurran) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md:71]: Fixed

File: tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md (hunk 2 above)
Comment (swcurran) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md:93]: It is a cause for concern -- as noted in the report.  Lots of use still, but no demand for updates, and so no ongoing activities.  The review of that PR is a few solid days of focused work, and the other maintainers on the project have not been able to dedicate time for that review.

**Review by tkuhrt — APPROVED** [review]

File: tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md (hunk 2 above)
Comment (tkuhrt) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md:93]: Should we consider moving this project to Dormant?

**Comment by swcurran** [comment]
I would definitely be against that. It is being used in a lot of places, and when things are needed, it will happen.  It’s just that it is so stable, that there is not a demand to update it.

@telegramSam @WadeBarnes

**Review by denyeart — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**